src/System.Management.Automation/engine/MshCommandRuntime.cs

```diff
@@ -2767,7 +2767,7 @@ private void DoWriteError(object obj)
             ActionPreference preference = pair.Value;
             if (errorRecord == null)
             {
-                throw PSTraceSource.NewArgumentNullException("errorRecord");
```

Comment:
Why not `NewArgumentNullException(nameof(errorRecord))`?

Reply:
Method signature is `DoWriteError(object obj)`, `errorRecord` is not an argument.

Reply:
Intention is to trace. We shouldn't remove this.

Reply:
We should not pass `"errorRecord"` as `paramName` as it is not an argument for this method.

Reply:
The question is not for the style PR.